src/azul/indexer/aggregate.py

```diff
@@ -256,6 +266,7 @@ def accumulate(self, value) -> None:
             self.value[value] += 1
 
     def get(self) -> list[Any]:
+        self.dropped = max(0, len(self.value) - self.max_size)
```

Comment:
It would be nice to cover `dropped` in doctests. I understand that this would require adding doctests to many accumulators but I think it is worth it.